{"name": "test_login_002", "status": "failed", "statusDetails": {"message": "AssertionError: assert 0", "trace": "self = <scripts.test_login.TestLogin object at 0x000001C95F4A9470>\n\n    def test_login_002(self):\n>       assert 0\nE       assert 0\n\nscripts\\test_login.py:7: AssertionError"}, "start": 1593341612870, "stop": 1593341612870, "uuid": "9faa8686-867a-45a5-80b7-cd9401e6db1b", "historyId": "51f4ddf5a78e0a1a297b81437f42fd9f", "testCaseId": "807d7923429a723b53f08fbc31059c73", "fullName": "scripts.test_login.TestLogin#test_login_002", "labels": [{"name": "parentSuite", "value": "scripts"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-IQ49U49"}, {"name": "thread", "value": "18284-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "scripts.test_login"}]}